doc/design.txt

```diff
@@ -272,7 +272,7 @@ purposes; they are not actually present in the boot vector.
     -----------------+--------+--------'
      swap: none                        |
     -----------------------------------'
-    
```

Comment:
> Further, "State I" and "State V" here both lead to the "none" swap type, which is two "states", not "one additional state" as you wrote in the above comment.

Right, "State I" and "State V" are really **two** states in the flash. But both of them end up being `NONE` in the code.